### DISCUSSION TITLE
Is there a way to insert new memo at the top of the existing list?

### DISCUSSION THREAD

**Discussion by mystcreater** [Quorafind/Obsidian-Thino#328] [Q&A 问答]
Example:

I have this existing entry:
# Journal
14:56 Bla bla bla

And I want if I enter a new memo to be added above the first one like this:
# Journal
15:12 Second bla bla bla
14:56 Bla bla bla

Thanks

**Comment by juestchaos** [comment]
By using the Pin feature, can it meet your use case
